citar-file.el

```diff
@@ -172,7 +187,7 @@ with citekey as filename.
 Returns the filename whether or not the file exists, to support a
 function that will open a new file if the note is not present."
   (let* ((possible-files
-          (citar-file--possible-names key dirs extensions))
+          (citar-file--possible-names key dirs extensions nil citar-file-find-additional-files))
```

Comment:
```suggestion
          (citar-file--possible-names key dirs extensions))
```
See my first comment above; it seems simpler to not add more function arguments up the chain and instead let-bind the defcustom in `citar--format-candidates`.